renderers/chord_diagram.spec.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const rewire = require('rewire');
+const chordDiagramJs = rewire('./chord_diagram.js');
```

Comment:
If you wanted to test `renderChordDiagram` as well, you can actually set `window` with rewire, which should get the tests working.

```javascript
chordDiagramJs.__set__('window', window);
```

Reply:
I gave it a try, but I got an error about createElementNS doesn't exist on undefined or something like that. In any case, I think tests that assert things about the SVG is likely to be less useful because the tests will always fail when you change the image. Probably best to use a human to test.